README.md

```diff
@@ -21,10 +21,10 @@ Features
 * Modern C99 syntax and a clean code layout
 * Deflate medium and quick algorithms based on Intels zlib fork
 * Support for CPU intrinsics when available
-  * Adler32 implementation using SSSE3, AVX2, Neon & VSX
+  * Adler32 implementation using SSSE3, AVX2, Neon, VMX & VSX
   * CRC32-B implementation using PCLMULQDQ & ACLE
   * Hash table implementation using CRC32-C intrinsics on x86 and ARM
-  * Slide hash implementations using SSE2, AVX2, Neon & VSX
+  * Slide hash implementations using SSE2, AVX2, Neon, VMX & VSX
   * Compare256/258 implementations using SSE4.2 & AVX2
   * Inflate chunk copying using SSE2, AVX2 & Neon
```

Comment:
Is it possible to add the new `WITH_VMX` and `--without-altivec` and `--with-power8` to the Advanced Options table in the readme?

Reply:
Done.